src/constants/tech-stack.ts

```diff
@@ -7,5 +7,9 @@ export const TECH_STACK = [
     'Tailwind CSS',
     'Supabase',
     'Discord.py',
+    'Requests',
+    'Vite',
+    'Pragmatic Drag & Drop',
+    'NextUI',
```

Comment:
Could you order this alphabetically? I know the current list isn't either but it'll be nicer if the entire list was ordered.